var/spack/repos/builtin/packages/py-prettytable/package.py

```diff
@@ -12,9 +12,13 @@ class PyPrettytable(PythonPackage):
     appealing ASCII tables.
     """
 
-    homepage = "https://code.google.com/archive/p/prettytable/"
-    pypi = "prettytable/prettytable-0.7.2.tar.gz"
+    homepage = "https://github.com/jazzband/prettytable"
```

Comment:
This should be done with `spack augment` to generate a package in `repo-patches`. This `builtin` repository will be reset to upstream's default on any conflict.

Reply:
Ok i see, i found the corresponding doc and i will follow it.

Reply:
> Can you write to this repository? Please let me know if not. I think the CI only runs from PRs within the same repository.

You mean if I can write in the repo-patches repo ?

Reply:
I meant if you can push a branch to this repository. For security reasons, it's simpler to restrict who can write to this repository and only execute CI for branches within the BlueBrain organization.

Reply:
Yes i tried and it seems that I can push commits into this repository

Reply:
Just seen that the last commit did not contained the good changes, I redo this quickly